thoth/common/logging.py

```diff
@@ -247,7 +247,7 @@ def init_logging(
                 _IGNORED_EXCEPTIONS.append((exc_module, exc_name))
             else:
                 root_logger.error(
-                    "Configuration for exceptions to be ignored not set correctly.")
+                    "The following exception couldn't be parsed - "+exception)
```

Comment:
Use lazy evaluation of the string:

```python
"The following configuration for ignoring exception couldn't be parsed: %r ", exception
```

Reply:
Does this only help in sentry grouping? Or is it because it has a better performance?

Reply:
Performance - the string is formatted based on log level.